### ISSUE TITLE
Got errors "Failed to load resource: the server responded with a status of 400 ()" and Refused to display '..' in a frame because it set 'X-Frame-Options' to 'SAMEORIGIN'.

### ISSUE THREAD

**Issue by basebasket** [be-cloud-js/open-erp-modules#6]
I 'm facing these errors when loading a form view likes employee or products and it doesn't show any thing when click on 'From GDrive...' under 'Attachment(s)'. I followed all steps in readme file and lost hours to find what am I missing but no luck, do you know how can I resolve these errors ?

**Comment by letzdoo-js** [comment]
Hi,

Have you enabled the Drive and the Picker API ?

Picker is new, I will update the Readme

Setting this up is a bit tricky, I don't see how to make it more simple, any suggestion is welcome.

**Comment by basebasket** [comment]
I enabled Drive API but can't find Picker API in my library, do I need config or enable something for Picker API ?

**Comment by letzdoo-js** [comment]
![capture d ecran 2016-11-25 a 12 29 13](https://cloud.githubusercontent.com/assets/12003829/20624032/d3d79a46-b30a-11e6-9755-40dfffda66f8.png)

**Comment by basebasket** [comment]
tks, I enabled it but errors still appear

**Comment by letzdoo-js** [comment]
the errors are "normal", it is google code that make use of this parameter, but it does not impact the behavior.

do you have a popup blocker ? this must be disabled as well.

are you using entrerprise or community version ? 9.0 ? which browser ?

**Comment by basebasket** [comment]
sorry, I comeback from 2 days absence . I use adblock extension of chrome, I stopped it and test again but no luck. My odoo version is community 9.0

**Comment by basebasket** [comment]
The errors was gone when I change line 50 in gdrive.js file **'immediate': true** to **'immediate': false** then change url config in google console from **http://0.0.0.0:8069/** to **http://localhost:8069/** , adding **/oauth2callback** after redirect URIs and open odoo with **http:/localhost:8069/** instead of **http://0.0.0.0:8069/**
![screenshot from 2016-11-28 11 06 04](https://cloud.githubusercontent.com/assets/12761212/20656116/d11baad8-b55b-11e6-9e03-06c11761b190.png)

Hope to see you update these in source and readme file

**Comment by letzdoo-js** [comment]
Hi,

This is good news. Using *http://0.0.0.0:8069/ <http://0.0.0.0:8069/>* is
not correct, I could say this in the readme.

For the immediate, I will make an update as soon as I ve got some time. The
immediate = false, which prevent a popup blink, does not work in Enterprise
either.

Will do that.

Jerome




http://www.be-cloud.be

**Comment by basebasket** [comment]
no, I means is using http://localhost:8069 instead of htpp://0.0.0.0:8069 because we usually use url http://0.0.0.0:8069 in local environment

Hope to see your update soon

**Comment by basebasket** [comment]
I've ran the module on v9 enterprise and It 's working well as expected, but on v10 community I got strange error when click on **From GDrive**. Do you have any plan for develop the module on v10 ?


![screenshot from 2017-01-18 09 50 33](https://cloud.githubusercontent.com/assets/12761212/22049091/c9475e44-dd63-11e6-81ec-c08f7f27e334.png)

**Comment by letzdoo-js** [comment]
Hi,
As soon as I get some time I will migrate it. Do not hesitate to let me know if you get a work around.

Best regards

Jerome

**Comment by chaosbunker** [comment]
+1 

Hi Jerome, just wanted to add my voice. I hope you will be able to fix the error, it would be awesome if the addon worked with 10.0! :-)

**Comment by hongquangminh** [comment]
Hi,
This issue fixed?
